gradio/blocks.py

```diff
@@ -881,14 +884,15 @@ def reverse(text):
                 "The `enable_queue` parameter has been deprecated. Please use the `.queue()` method instead.",
                 DeprecationWarning,
             )
-        if self.is_space and self.enable_queue is None:
-            self.enable_queue = True
-        if self.enable_queue is None:
-            self.enable_queue = False
-        utils.run_coro_in_background(self.create_limiter, max_threads)
+        if self.is_space:
+            self.enable_queue = enable_queue is not False
+        else:
+            self.enable_queue = enable_queue is True
```

Comment:
This logic felt incorrect, hence changed it.

Reply:
Now that I think about it, it was not incorrect :D

Reply:
You're going to undo this change right?

Reply:
Nope, since it works like this as well :)

Reply:
You're right! I think the old way was easier to understand though 🙈